src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -18,13 +16,11 @@
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a module to the GradPad. "
             + "Parameters: "
-            + PREFIX_CODE + "MODULE CODE "
-            + PREFIX_CREDITS + "MODULAR CREDITS "
+            + "MODULE CODE "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_CODE + "CS1231 "
-            + PREFIX_CREDITS + "4 "
-            + PREFIX_TAG + "core ";
+            + "CS1231 "
+            + PREFIX_TAG + "boring";
```

Comment:
```suggestion
            + PREFIX_TAG + "Foundation";
```
Hehe maybe change to actual tag examples

Reply:
oh no i didn't see this, will change!